components/menu/index.css

```diff
@@ -10,170 +10,676 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+    ".            .                      .             .        descriptionArea  .         .           .                 "
+    ".            .                      .             .        submenuArea      .         .           .                 ";
+  
+  grid-template-columns: auto auto auto auto 1fr auto auto auto;
+  grid-template-rows: 1fr auto;
+}
 
-  min-block-size: var(--spectrum-listitem-texticon-height);
+.spectrum-Menu-item .spectrum-Menu-item {
+  display: grid;
+  grid-template-areas: "submenuItemLabelArea submenuItemValueArea";
+  grid-template-columns: auto auto;
+  grid-template-rows: 1fr auto;
+}
 
-  font-size: var(--spectrum-listitem-texticon-text-size);
-  font-weight: var(--spectrum-listitem-texticon-text-font-weight);
-  font-style: normal;
-  text-decoration: none;
+.spectrum-Menu-item .spectrum-Menu-item .spectrum-Menu-itemLabel {
+  grid-area: submenuItemLabelArea;
+}
+
+.spectrum-Menu-item .spectrum-Menu-item .spectrum-Menu-itemValue {
+  grid-area: submenuItemValueArea;
+}
+
+.spectrum-Menu-item:focus {
+  outline: none;
+
+  .spectrum-Menu-itemLabel {
+    color: var(--highcontrast-menu-item-label-content-color-focus, var(--mod-menu-item-label-content-color-focus, var(--spectrum-menu-item-label-content-color-focus)));
+  }
+
+  .spectrum-Menu-sectionHeading {
+    color: var(--highcontrast-menu-section-header-color, var(--mod-menu-section-header-color, var(--spectrum-menu-section-header-color)));
+  }
+
+  .spectrum-Menu-itemDescription {
+    color: var(--highcontrast-menu-item-description-color-focus, var(--mod-menu-item-description-color-focus, var(--spectrum-menu-item-description-color-focus)));
+  }
+
+  .spectrum-Menu-itemValue {
+    color: var(--highcontrast-menu-item-value-color-focus, var(--mod-menu-item-value-color-focus, var(--spectrum-menu-item-value-color-focus)));
+  }
+  
+  .spectrum-Menu-itemIcon:not(.spectrum-Menu-chevron, .spectrum-Menu-checkmark) {
+    fill: var(--highcontrast-menu-item-label-icon-color-focus, var(--mod-menu-item-label-icon-color-focus, var(--spectrum-menu-item-label-icon-color-focus)));
+  }
+
+  .spectrum-Menu-chevron {
+    fill: var(--highcontrast-menu-collapsible-icon-color, var(--mod-menu-collapsible-icon-color, var(--spectrum-menu-collapsible-icon-color)));
+  }
+
+  .spectrum-Menu-checkmark {
+    fill: var(--highcontrast-menu-checkmark-icon-color-focus, var(--mod-menu-checkmark-icon-color-focus, var(--spectrum-menu-checkmark-icon-color-focus)));
+  }
+}
+
+.spectrum-Menu-item:focus-ring,
+.spectrum-Menu-item.is-focused {
+  background-color: var(--highcontrast-menu-item-background-color-key-focus, var(--mod-menu-item-background-color-key-focus, var(--spectrum-menu-item-background-color-key-focus)));
+  border-color: var(--highcontrast-menu-item-focus-indicator-color, var(--mod-menu-item-focus-indicator-color, var(--spectrum-menu-item-focus-indicator-color)));
+
```

Comment:
Needs
`outline: none;`

Combined with adding :focus-visible as noted above, this gives me (same results in Chrome and Firefox):
<img width="192" alt="Screen Shot 2022-09-23 at 4 23 19 PM" src="https://user-images.githubusercontent.com/25614178/192051059-04ded8be-432d-4c9c-bed9-2b70354d6f74.png">